integrations/uber.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Uber",
+  "auth": {
+    "authorizationURL": "https://login.uber.com/oauth/v2/authorize",
+    "tokenURL": "https://login.uber.com/oauth/v2/token",
+    "authType": "OAUTH2",
+    "tokenParams": {},
+    "authorizationParams": { "response_type": "code", "grant_type": "authorization_code" }
+  },
+  "request": {
+    "baseURL": "https://api.uber.com/v1.2",
+    "headers": {
+      "Accept": "application/json",
+      "Authorization": "Bearer ${auth.accessToken}",
+      "User-Agent": "Pizzly (https://github.com/bearer/pizzly, 0.2.0)"
```

```suggestion
      "User-Agent": "Pizzly"
```

While I presume you had something great in mind to add both a link to the repo and the version number, sounds better to keep this header simple 🙏  Others configurations are using the proposed change.

---

In the issue #144 description, there is an example PR(Discord API) which has link to the repo and the version number. So, I assumed that is standard. Anyways made the suggested change. Thanks